VirtualList/tests/VirtualList-specs.js

```diff
@@ -94,170 +55,151 @@ describe('VirtualList', () => {
 		expect(actual).toBe(expected);
 	});
 
-	test('should render (clientHeight / itemHeight + overhang) items', () => {
+	test('should render overhang items when clientSize is not given', () => {
```

Comment:
I'm confused. Why are 3 items rendered?

Reply:
I am not sure, but in the case of VirtualList, There are 3 overhang items.
So I guess only overhang items seem to be drawn.

Reply:
🤔